src/main/java/practice/StreamPractice.java

```diff
@@ -14,16 +19,33 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(word -> word.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
You can do it in one line
```suggestion
                .flatMap(word -> Stream.of(word.split(",")))
```